### ISSUE TITLE
Can the eve_universe DB be shared among multiple instances?

### ISSUE THREAD

**Issue by patrickebates** [goryn-clade/pathfinder#237]
Pretty much the title.  I'm still researching on this myself, but I'm wondering if the eve_universe DB is something that can be shared with multiple Pathfinder instances in the same MySQL server in order to save space.  So far, it appears to be pretty much static data to me.

**Comment by TyrHeimdalEVE** [comment]
Yes, as long as you give all the Pathfinder instances access to the `eve_universe` database, then it can be used.

Each instance would need to reside on the same MySQL server as there is no separate MySQL configuration for this specific DB.
